docs/gameplay-sessions/gameplay-sessions-usage.mdx

```diff
@@ -16,24 +16,48 @@ Sessions.
 ## Recording Gameplay
 
 Once the overlay has been added to your scene(s), click play.
-The Regression Games logo at the bottom of the Scene View will display recording and playback controls.
+The Regression Games toolbar at the bottom of the Scene View will display recording and playback controls.
 
-Recording can be started by clicking the red "record" button below the logo.
+Recording can be started by clicking the red "record" button on the right side of the toolbar.
 Once clicked, the SDK will begin recording your gameplay. You can tell that recording is active when the red button begins pulsing.
 Clicking the button again will stop the recording and sync the Gameplay Session to your Regression Games account.
 
 ![Start a recording](img/record.png)
 
 ## Playing Back a Recording
 
-A recording can be replayed by initiating "play" mode from your Unity editor and selecting the menu button below the Regression Games logo.
+### Loading a Recording
+A recording can be replayed by initiating "play" mode from your Unity editor and selecting the menu button on the Regression Games toolbar.
 This will launch a file explorer at the location of the recordings folder on your device.
 Navigate into the folder of the recording you'd like to replay, select its `data.zip` file, and then click "Load Replay".
-Once you've selected a replay, click the Play button below the Regression Games logo to begin playback.
 
 ![Open the Gameplay Session Selector](img/overlay_click_load.png)
 ![Select the Gameplay Session](img/overlay_choose_session.png)
 
+### Playing a Recording
+Once you've selected a replay, click either the Loop or Play button below on the Regression Games toolbar to begin playback.
+- The Loop function will continue to play the recording over and over until a failure occurs.
```

Comment:
We may want to mention that the recording needs to be able to get back to an initial state on its own, ie our SDK is not responsible for "resetting" the game to a valid state between loops